props/saber_fett263_buttons.h

```diff
@@ -1911,6 +1923,45 @@ SaberFett263Buttons() : PropBase() {}
     sound_library_.SayBool(current_value);
   }
 
+  int GetStyleNumber(int blade_num) {
+    const char* tmp;
+    tmp = current_preset_.GetStyle(blade_num);
+    int style_num = FirstWord(tmp, "builtin") ? atoi(SkipWord(tmp)) : 0;
+    return style_num;
+  }
+
+  void ChangeStyleNumber(int blade_num, int direction) {
+    int num_presets = current_config->num_presets;
+    style_num_ += direction;
+    if (style_num_ < 0) style_num_ = num_presets - 1;
+    if (style_num_ >= num_presets) style_num_ = 0;
+    char style_arg[10];
+    itoa(style_num_, style_arg, 10);
+    current_preset_.SetStyle(blade_num, style_parser.SetArgument(current_preset_.GetStyle(blade_num), 1, style_arg));
+  }
+
+  void ChangeStyleNumberAllBlades(int direction) {
+    for (int i = 1; i <= NUM_BLADES; i++) {
+      style_num_ = GetStyleNumber(i);
+      ChangeStyleNumber(i, direction);
+    }
+    current_preset_.Save();
+    UpdateStyle();
+  }
+
+  void ChangeFont(int direction) {
```

Comment:
This needs a comment to explain that it uses font_num_, not whatever the "current" font for this preset is.

Reply:
added